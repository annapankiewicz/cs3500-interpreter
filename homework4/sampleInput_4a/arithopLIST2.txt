x = 10 + (list(1, 2))
