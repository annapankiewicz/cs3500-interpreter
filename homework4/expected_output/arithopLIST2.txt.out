Line 2: Arg 2 cannot be function or null or list
